net-analyzer/prelude-lml-rules/prelude-lml-rules-5.1.0.ebuild

```diff
@@ -0,0 +1,21 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Prelude LML community ruleset"
+HOMEPAGE="https://www.prelude-siem.org"
+SRC_URI="https://www.prelude-siem.org/pkg/src/${PV}/${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+BDEPEND="~net-analyzer/prelude-lml-${PV}"
```

Comment:
I've explained the roles of `RDEPEND`, `DEPEND` and `BDEPEND` in a previous comment. Please fix this accordingly. Let me know if there are any confusions.

Reply:
done